src/js/list_wordles.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+/* global inverted_wordles */
+
+inverted_wordles.listWordles = function (wordles, wordlesAreaSelector) {
```

Comment:
Name is a bit misleading - it actually renders all the wordles received from the API. If the appendNewWordleRow were reorganised a bit, it could be used here.